app/services/panoptes_api.rb

```diff
@@ -9,8 +9,15 @@ class PanoptesApi
            :authenticated_user_display_name,
            :token_expiry, to: :client
 
-  def initialize(token)
-    @token ||= token
+  def initialize(token:, admin:)
```

Comment:
Not blocking - fwiw when we start passing in boolean values to change run time behaviour this is a flag for a different class to me. I.e. create another `ClientCredentialsPanotpesApi.new()` class that sets up the auth values correctly and call that instead of using the `:admin=true`

Reply:
great idea, easily done. 👍